pages/platform/databases/cassandra_01_capabilities/guide.de-de.md

```diff
@@ -105,6 +105,13 @@ Here are the node types you can choose from:
 
 Right now, all nodes of a given cluster should be of the same type and live in the same region.
 
+#### Disk type
+
+The type of storage available may vary according to the region your cluster lives in. Thus, your cluster may be backed by e.g. *High Speed* or *High Speed Gen2* block storage.
```

Comment:
Add this link for customer to see performance differences between technology: https://www.ovhcloud.com/fr/public-cloud/block-storage/